Sources/Components/SaveSearchView/SearchListEmptyView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+// MARK: - SearchListEmptyViewDelegate
+
+public protocol SearchListEmptyViewDelegate: AnyObject {
+    func searchListEmptyViewDidSelectActionButton(_ searchListEmptyView: SearchListEmptyView, for state: SearchListEmptyView.ViewState)
+}
+
+@objc public class SearchListEmptyView: UIView {
+
+    // MARK: - Public properties
+
+    public enum ViewState: Int {
+        case initial = 0
```

Comment:
I think it starts with zero by default.

Reply:
BTW, does this `ViewState` need to be in `FinniversKit`? It doesn't seem to be used anywhere in this view, so perhaps it could be stored in the app. What do you think?

Reply:
I just thought it was more messy to add it to the controller instead, since the swift extension can't contain stored properties, I would have to add it to objc and try to make that work, which is probably possible. It might be worth the time?

Reply:
I see. No, I don't think it's worth it, it would be better to refactor that objc controller to Swift one day. But for now, let's keep it in `FinniversKit` 😉

Reply:
True 😄